share/spack/gitlab/cloud_pipelines/configs/linux/x86_64_v1

```diff
@@ -0,0 +1 @@
+x86_64
```

Comment:
Is there such a thing as `x86_64_v1`?  I thought it was just `x86_64`.

Reply:
I think so. I was mostly just copy-pasting to cover my bases. I came up with a new strategy though since the auto detection wasn't working reliably.